docs/topics/proc_creating-a-vertx-application.adoc

```diff
@@ -0,0 +1,204 @@
+[#creating-a-vertx-application]
+= Creating an application
+
+Create a simple `Greeting` application to run on OpenShift using {Vertx}.
+The following procedure shows you how to:
+
+* Write some simple application code that makes use of functionalities provided by {Vertx}.
+* Declare dependencies and configure the application build using a `pom.xml` file.
+* Start your application on localhost and verify that it works.
+
+
+.Prerequisites
+
+* Maven installed.
+
+.Procedure
+
+. Create the application directory and navigate to it.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ mkdir myApp
+$ cd myApp
+----
+
+. Create a `pom.xml` file.
++
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <name>My Application</name>
+  <description>Example application using RHOAR Vert.x</description>
```

Comment:
By convention (https://maven.apache.org/developers/conventions/code.html#POM_Code_Convention), the `<name>` and `<description>` should be _after_ groupId/artifactId/version/packaging.

Reply:
@Ladicek Thank you, I wasn't sure where to place it. I'll move the `<name>` and `<description>` so that the example conforms to the convention:)